force-app/main/default/lwc/scheduler_drag_from_grid/scheduler_drag_from_grid.css

```diff
@@ -0,0 +1,158 @@
+.container {
+  height: 40em;
+}
+
+.b-float-root {
+  z-index: 100;
+}
+
+.b-sch-event {
+  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.25);
+}
+
+.container {
+  display: flex;
+  flex: 1 1 100%;
+  flex-direction: row;
+}
+.container:not(.b-side-by-side) .b-schedulerpro {
+  order: 0;
+}
+.container:not(.b-side-by-side) .b-splitter {
+  order: 1;
+}
+.container:not(.b-side-by-side) .b-unplannedgrid {
+  order: 2;
+}
+.container.b-side-by-side .b-grid-header {
+  height: 57px;
+}
+.container.b-side-by-side .b-unplannedgrid {
+  flex: 0 1 200px;
+}
+
+.b-toolbar {
+  height: 3em;
+}
```

Comment:
These lines suggest this CSS is from a different demo. Please rebuild examples sass and copy app.css from Scheduler/examples/dragfromgrid demo (or take it online)